### ISSUE TITLE
Purge `godotcontrollerdb.txt`

### ISSUE THREAD

**Issue by offalynne** [godotengine/godot-proposals#10540]
### Describe the project you are working on

N/A

### Describe the problem or limitation you are having in your project

Gamepad mappings a mess on web target

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

godotcontrollerdb.txt is superfluous and should not be external data. The only significant content consists of "web" mappings and these are a disaster: web target should *always* conform to the standard mapping provided by the browser. Missing device support and related issues on web target should *always* flow upstream to the relevant project (Chromium, Mozilla). Overriding the standard mapping is never worthwhile, and will only cause per-platform conflicts and future device compatibility issues. If Godot users find problems with an incorrect or missing standard mapping for a given device in their browser, they should report these issues to the relevant browser project instead of creating a future problem for Godot compatibility.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

N/A

### If this enhancement will not be used often, can it be worked around with a few lines of script?

N/A

### Is there a reason why this should be core and not an add-on in the asset library?

N/A

**Comment by thygrrr** [comment]
Agree, this is why some builds of some games on some browsers suddenly have inverted controls etc.
